### PR TITLE
Bug 1700121 Add sleep task to burnham DAG

### PR DESCRIPTION
**Pull request by jklukas**
Per https://bugzilla.mozilla.org/show_bug.cgi?id=1700121#c10

### CODE REVIEW

File: dags/burnham.py [hunk 1]
```diff
@@ -518,6 +519,30 @@ def encode_test_scenarios(test_scenarios):
...
 
 
+def do_sleep(minutes):
+    """Sleep for the given number of minutes.
+
+    Writes out an update every minute to give some indication of aliveness.
+    """
+    print(f"Configured to sleep for {minutes} minutes. Let's begin.")
+    for i in range(minutes, 0, -1):
+        print(f"{i} minute(s) of sleeping left")
```
Comment (hackebrot) [inline dags/burnham.py:529]: Do you generally recommend `print` commands over logging in Airflow operators?

File: dags/burnham.py [hunk 2]
```diff
@@ -518,6 +519,30 @@ def encode_test_scenarios(test_scenarios):
...
+        time.sleep(60)
+
+
+def sleep_task(minutes, task_id):
+    """Return an operator that sleeps for a certain number of minutes.
+
+    :param int    minutes: [Required] Number of minutes to sleep
+    :param string task_id: [Required] ID for the task
```
Comment (hackebrot) [inline dags/burnham.py:537]:
```suggestion
    :param string task_id: [Required] ID for the task
    
    :return: PythonOperator
```

**Review by hackebrot — CHANGES_REQUESTED** [review]
Looks good to me! I added one general question and one suggestion. 📝

File: dags/burnham.py (hunk 1 above)
Comment (jklukas) [reply dags/burnham.py:529]: Logging rather printing is a very good idea 👍

**Review by hackebrot — APPROVED** [review]
LGTM! 👩🏻‍🚀

**Pull request merged**